docker/qgisserver/geomapfish_qgisserver/accesscontrol.py

```diff
@@ -45,14 +44,20 @@ def __init__(self, server_iface):
         super().__init__(server_iface)
 
         self.server_iface = server_iface
+        self.initialized = False
 
         try:
             config.init(os.environ.get("GEOMAPFISH_CONFIG", "/etc/qgisserver/geomapfish.yaml"))
 
-            c2cwsgiutils.broadcast.init(None)
+            c2cwsgiutils.broadcast.init()
 
             configure_mappers()
-            engine = sqlalchemy.create_engine(config.get("sqlalchemy_slave.url"))
+            QgsMessageLog.logMessage(
+                "Connect to the database: {}".format(config.get("sqlalchemy_slave.url")),
```

Comment:
The password is part of the URL => not a good idea to log as is

Reply:
+1, thanks :-)